test/Syntax/Inputs/serialize_multiple_decls.json

```diff
@@ -17,18 +17,10 @@
                       "kind": "StructDecl",
                       "layout": [
                         {
-                          "kind": "AttributeList",
```

Comment:
Big fan of the JSON space savings this wins us! Hopefully the Codable changes to `RawSyntax.swift` aren't too crazy, but I absolutely am a fan of avoiding serializing the contents of missing nodes altogether.